src/executables/util_cs.c

```diff
@@ -3212,6 +3215,10 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 
   check_replica_info = (HA_GET_MODE () == HA_MODE_REPLICA);
   pageid = utility_get_option_bigint_value (arg_map, APPLYINFO_PAGE_S);
+  if (((pageid != APPLYINFO_DEFAULT_LOG_PAGEID) && (log_path == NULL)) || (pageid < 0))
```

Comment:
Jooho suggested [this idea](https://github.com/CUBRID/cubrid/pull/3374#discussion_r840109775) and I decided to choose `APPLYINFO_DEFAULT_LOG_PAGEID` instead of `NULL_LOG_PAGEID`. 
Because `NULL_LOG_PAGEID` is already exists.

Reply:
1. can you send me message about this task in teams app?
2. This is for the usage check.

Manual is something weird. Actually, I cannot understand what is the 'active' page. So I try to solve as the testcase.

Reply:
> Jooho suggested https://github.com/CUBRID/cubrid/pull/3374#discussion_r840109775 and I decided to choose APPLYINFO_DEFAULT_LOG_PAGEID instead of NULL_LOG_PAGEID.
Because NULL_LOG_PAGEID already exists.

My suggestion was to use the existing 'NULL_LOG_PAGEID', not to define a new one.

Reply:
> 1. can you send me message about this task in teams app?

OK, let's have a talk next week.

> 2. This is for the usage check.

When `log_path == NULL`, why should we check the value of it for usage?